src/components/screens/SendScreen/screens/TransactionAmountScreen.tsx

```diff
@@ -220,7 +264,26 @@ const TransactionAmountScreen: React.FC<TransactionAmountScreenProps> = ({
         error instanceof Error ? error.message : String(error),
       );
     }
-  };
+  }, [
+    tokenAmount,
+    selectedBalance,
+    recipientAddress,
+    transactionMemo,
+    transactionFee,
+    transactionTimeout,
+    network,
+    publicKey,
+    buildTransaction,
+  ]);
+
+  useFocusEffect(
+    useCallback(() => {
+      if (hasOpenedAddMemoExplanationBottomSheet) {
+        handleOpenReview();
+        setHasOpenedAddMemoExplanationBottomSheet(false);
+      }
+    }, [hasOpenedAddMemoExplanationBottomSheet, handleOpenReview]),
+  );
```

Comment:
is this correct/necessary? looks like we are not setting `setHasOpenedAddMemoExplanationBottomSheet(true)` anywhere in this PR

Reply:
friendly reminder to remove this code

Reply:
Just noting here for tracking, I see you have removed this code on the other PR: https://github.com/stellar/freighter-mobile/pull/273/files#diff-9822eb8a445dde0926940107e4a8283b76617d3fc9f1fde28336f304acb87845L282-L290